packages/sdk/src/packageMetadata.ts

```diff
@@ -2,4 +2,4 @@
 // Licensed under the MIT license.
 
 export const name = "@microsoft/teamsfx";
-export const version = "0.3.0-alpha.5";
+export const version = "0.3.0-alpha.6";
```

Comment:
Not blocking this PR.
@tecton seems the auto version bump does not commit changes made to packageMetadata.ts?